### ISSUE TITLE
Not a Load gen or benchmarking tool

### ISSUE THREAD

**Issue by Fujio-Turner** [bentonam/fakeit#188]
People are confusing fake it as a load gen or benchmarking tool.

> "Why can I only put in XYZ documents per second into Couchbase with fake it."

On the Read.me file could you put a disclaimer that fakeit is a sample document generator not a load gen tool and create a link to Couchbase pillowfight for load gen.
https://blog.couchbase.com/performance-testing-load-testing-couchbase-pillowfight/

**Comment by bentonam** [comment]
@Fujio-Turner I agree, I've talked about it being a development tool. There are ways to generate millions/billions of records with FakeIt but it's a little convoluted at the present time.
